extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -238,12 +239,25 @@ export class ConsentUI {
    * @return {!Element}
    */
   createPlaceholder_() {
-    // TODO(@zhouyx): Allow publishers to provide placeholder upon request
     const placeholder = this.parent_.ownerDocument.createElement('placeholder');
     toggle(placeholder, false);
     const {classList} = placeholder;
-    classList.add(consentUiClasses.fill);
-    classList.add(consentUiClasses.placeholder);
+    classList.add(consentUiClasses.loadingPlaceholder);
+
+    const loadingSpinner = htmlFor(placeholder)`
+    <svg viewBox="0 0 40 40">
+      <defs>
+        <linearGradient id="grad">
+          <stop stop-color="rgb(105, 105, 105)"></stop>
+          <stop offset="100%"
+          stop-color="rgb(105, 105, 105)"
+          stop-opacity="0"></stop>
+        </linearGradient>
+      </defs>
+      <path d="M11,4.4 A18,18, 0,1,0, 38,20" stroke="url(#grad)"></path>
+    </svg>`;
```

Comment:
If we're using this definition in multiple places, it also might be a good idea to store it externally and import into this file.

Reply:
As of now, we are only thinking of using this here. Was asking @sparhami about this in the past, if we had a system in place for re-used html elements, and it appears that we do not. So we should probably do that as well if we continue to add more HTML literals.